modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1081,19 +1088,25 @@ class Edit_User extends \NDB_Form
                     = 'Please specify password or click Generate new password';
             }
         }
+        // Ensure that the password and confirm password fields match.
+        // TODO This validation should be done on the front-end instead.
```

Comment:
```suggestion
        // TODO This validation should also be done on the front-end.
```
Never rely to front-end check!

Reply:
I don't think it's really an issue in this case. It's just a double-check so that someone doesn't make a typo while entering their password.